packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1035,13 +1042,14 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @override
   void didUpdateWidget(T oldWidget) {
     super.didUpdateWidget(oldWidget);
-    if (widget.isAlwaysShown != oldWidget.isAlwaysShown) {
-      if (widget.isAlwaysShown == true) {
-        _maybeTriggerScrollbar();
-        _fadeoutAnimationController.animateTo(1.0);
-      } else {
-        _fadeoutAnimationController.reverse();
-      }
+    // If `isAlwaysShown` is true and does not change,
+    // it may be necessary to trigger a scroll event to show or hide the bar when the
+    // scrollable widget viewport size changed.
+    if (widget.isAlwaysShown == true) {
+      _maybeRequestEmptyScrollEvent();
```

Comment:
Apologies for the delay. I think this may be the culprit. I have been looking into the customer code. It has `isAlwaysShown: true`, but the test failure from this patch is happening when the `Scrollbar` is trying to trigger an empty scroll event and there is no position attached. 

Maybe here we can skip triggering if a position is not attached? Although we do not listen for changes like that, so we would not know when a position is later attached..

I've been trying to figure this out for a bit and I am not sure how to proceed. I thought maybe we could migrate the customer to provide Scrollbars via ScrollBehavior.buildScrollbar, as there should be a position attached by that point when the scrollable calls on the ScrollConfiguration to wrap with the Scrollbar, but I haven't had a chance yet to test that and see if it makes a difference. I'd like to try it, I should be able to give it a go soon. Thank you for your patience @xu-baolin! If you have any thoughts on this, let me know. :)

Reply:
Thank you very much for your feedback!
I am curious that the event was requested at the post-frame callback and the controller should have clients at this point.